src/number_theory/bernoulli.lean

```diff
@@ -139,8 +161,46 @@ begin
   rw ← cast_sub hk,
   congr',
   field_simp [show ((n - k : ℕ) : ℚ) + 1 ≠ 0, by {norm_cast, simp}],
-  -- down to nat
   norm_cast,
   rw [mul_comm, nat.sub_add_eq_add_sub hk],
   exact choose_mul_succ_eq n k,
 end
+
+open power_series
+open nat
+
+theorem bernoulli_power_series :
+  (power_series.mk (λ n, ((bernoulli n) / (nat.factorial n) : ℚ))) * (exp ℚ - 1) = X * (exp ℚ) :=
```

Comment:
Or even
```suggestion
  power_series.mk (λ n, (bernoulli n / nat.factorial n : ℚ)) * (exp ℚ - 1) = X * exp ℚ :=
```
since function call precedent is higher than all binary operators